### PR TITLE
Added unit methods for AI cluster tests

### PR DESCRIPTION
**Pull request by hzhao-github**
One failure from https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/6066/ is not caused by this change

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java [hunk 1]
```diff
@@ -396,9 +470,58 @@ public static Domain createDomainResourceWithAuxiliaryImageClusterScope(
...
+          replicaCount,
+          List.of(clusterName),
+          auxiliaryImagePathVolume,
+          auxiliaryImageDomainScopeNames,
+          auxiliaryImageClusterScopeNames);
+
+    return domainCR;
+  }
+
+  /**
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java:489]: re format the javadoc with a  starting *  on each line

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java [hunk 2]
```diff
@@ -298,9 +334,47 @@ public static Domain createDomainResource(
...
+   * @param encryptionSecretName name of the secret used to encrypt the models
+   * @param replicaCount number of managed servers to start
+   * @param clusterNames a list of the cluster name to add in domain
+   * @param auxiliaryImagePath auxiliary image path, parent location for Model in Image model and WDT installation files
+   * @param auxiliaryImageVolumeName auxiliary image volume name
+   * @param auxiliaryImageName image names including tags, image contains the domain model, application archive if any
+   *                   and WDT installation files
+   * @return domain object of the domain resource
+   */
+  public static Domain createDomainResource(
```
[the remark this answers is not in the document]
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java]: changed

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java (hunk 1 above)
Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java:489]: changed

**Comment by robertpatrick** [comment]
What is the status of this PR?  It is over 3 months old so is it still relevant?  If so, then update the PR to make it so that it can be merged with current code.  Otherwise, please close it.

**Pull request closed without merging**